Test/Publish-WhiskeyBuildMasterPackage.Tests.ps1

```diff
@@ -81,6 +109,8 @@ function Init
     $script:apiKey = $defaultApiKey
     $script:releaseId = $defaultReleaseId
     $script:taskParameter = @{ }
+    $script:deployToMap = @{ 'BranchName' = $defaultScmBranch; 'ReleaseName' = $releaseName }

```

Comment:
Should this default to no map?

Reply:
Yup. I waffled on this and chose this default to avoid having to configure a `GivenDeployTo` for all of the failing tests. But, I think I like the idea of a $null default instead. I'll update